src/app/ngrx/reducers/models/podcast-totals.model.ts

```diff
@@ -1,11 +1,22 @@
 import { Rank } from './rank.model';
+import { GroupType, GROUPTYPE_GEOSUBDIV } from './group.type';
+import * as dateFormat from '../../../shared/util/date/date.format';
```

Comment:
Not sure if this applies to our version of TypeScript, but it looks like you can now use a `baseUrl` configuration to optionally use absolute imports, which would be handy right here: http://www.typescriptlang.org/docs/handbook/module-resolution.html#base-url

Reply:
😎